#!/usr/bin/env python
PACKAGE = "my_repository"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("enabled",   bool_t,   0, "Enable_map",  True)
gen.add("object_x",    double_t,    0, "X coordinate of object position", 0.0,  -100.0, 100.0)
gen.add("object_y", double_t, 0, "Y coordinate of object position",    0.0, -100.0,   100.0)


exit(gen.generate(PACKAGE, "my_repository", "map_cfg"))
